andela/travel_tool_front
# #160371227 send notification for approved request

#### What does this PR do?
Sends notification to requesters if their manager approves their request. 

#### Description of Task to be completed?
- Set up socket.io for the client side
- Add listener receive notifications from the server and render it on the notification pane

#### How should this be manually tested?
- Clone this repo - git clone https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Check out this branch - git checkout ft-create-request-notification-160371133
- Install dependencies and start the server - make start
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/46)
- Create a new travel request
- Click on a request after routing to `My Approvals` page and approve the request
- A notification will be sent to you.
- Click on the `Bell icon on the navbar` to see the new notification


#### Any background context you want to provide?
Ensure to log out of the app and log back in after setting up the server using this [guide](https://github.com/andela/travel_tool_back/pull/46).

#### What are the relevant pivotal tracker stories?
[#160371227](https://www.pivotaltracker.com/story/show/160371227)

#### Screenshots (if appropriate)
None

#### Questions:
None
